From 8b865a1f2f35f591e8004a75765bd9b3133229a5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Fri, 29 Aug 2014 11:29:36 +0100
Subject: [PATCH] sigh, simply forgot to include the endian header

so the ppc64 bridge was broken

Change-Id: Ib60274ed25531f500bcd97f5ea9b04091ce01825
(cherry picked from commit 7da6716e98f22d6cb968c3218b5f4936f868a977)
---
 bridges/source/cpp_uno/gcc3_linux_powerpc64/cpp2uno.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bridges/source/cpp_uno/gcc3_linux_powerpc64/cpp2uno.cxx b/bridges/source/cpp_uno/gcc3_linux_powerpc64/cpp2uno.cxx
index eaee739..acee17d 100644
--- a/bridges/source/cpp_uno/gcc3_linux_powerpc64/cpp2uno.cxx
+++ b/bridges/source/cpp_uno/gcc3_linux_powerpc64/cpp2uno.cxx
@@ -21,7 +21,7 @@
 #include <com/sun/star/uno/genfunc.hxx>
 #include <uno/data.h>
 #include <typelib/typedescription.hxx>
-
+#include <osl/endian.h>
 #include "bridges/cpp_uno/shared/bridge.hxx"
 #include "bridges/cpp_uno/shared/cppinterfaceproxy.hxx"
 #include "bridges/cpp_uno/shared/types.hxx"
-- 
1.9.3

